samples/experimental/stack.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: experimental.kpack.pivotal.io/v1alpha1
+kind: Stack
+metadata:
+  name: bionic-stack
+spec:
+  id: "io.buildpacks.stacks.bionic"
+  buildImage:
+    image: "cloudfoundry/build:base-cnb"
```

Comment:
I wonder if we should put the digest here in the samples?

Reply:
instead of the tags for the images?